From baf9ae9038dba56324e08e5df0023225a6067154 Mon Sep 17 00:00:00 2001
From: Rui Ueyama <ruiu@cs.stanford.edu>
Date: Tue, 16 Jul 2024 11:59:22 +0900
Subject: [PATCH] Fix a test on Debian

If the default linker doesn't complain, just skip the test.

Fixes https://github.com/rui314/mold/issues/1301
---
 test/elf/arm_abs-error.sh | 2 ++
 1 file changed, 2 insertions(+)

Index: mold/test/elf/arm_abs-error.sh
===================================================================
--- mold.orig/test/elf/arm_abs-error.sh
+++ mold/test/elf/arm_abs-error.sh
@@ -12,5 +12,7 @@ extern char foo;
 int main() { printf("foo=%p\n", &foo); }
 EOF
 
+$CC -o $t/exe -pie $t/a.o $t/b.o >& /dev/null && skip
+
 ! $CC -B. -o $t/exe -pie $t/a.o $t/b.o >& $t/log
 grep -q 'recompile with -fPIC' $t/log
